snapshot-labs/sx-monorepo
# feat: add "Vote" button to single choice voting form

### Summary



Closes: #252

Add a VOTE button for single choice voting type, instead of submitting the vote on choice click

![Screenshot 2024-04-06 at 15 41 24](https://github.com/snapshot-labs/sx-monorepo/assets/495709/b02eb8cf-6440-4119-a62a-3d3e56fad24a)


### How to test

1. Go to http://localhost:8080/#/s:pistachiodao.eth/proposal/0x02c3fcd64e86157d07c88e5a715ac08f57655917f8bfd5be30a99092136511ec
2. It should show a vote button
3. Clicking on a choice should select the choice
4. Clicking on the vote button should submit the vote
5. The VOTE button should be disabled when not choices are selected
